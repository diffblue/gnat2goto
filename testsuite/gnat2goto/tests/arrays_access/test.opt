ALL XFAIL cbmc crashes when reading gnat2goto output (in symex)
