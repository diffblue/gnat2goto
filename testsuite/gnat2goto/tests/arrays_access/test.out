[arrays_access.assertion.1] line 5 Ada Check assertion: SUCCESS
[arrays_access.assertion.2] line 6 assertion Actual7(1) = 2: SUCCESS
VERIFICATION SUCCESSFUL
